.gitignore

```diff
@@ -14,7 +14,6 @@
 .deploy_info
 .ruby-gemset
 .ruby-version
-.tool-versions
```

Comment:
I wonder why removing this. Do you want to support it officially?

Reply:
it is a redundant entry, we need it only once